readthedocs/projects/views/mixins.py

```diff
@@ -44,3 +47,46 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context[self.project_context_object_name] = self.get_project()
         return context
+
+
+class ProjectImportMixin:
+
+    """Helpers to import a Project."""
+
+    def import_project(self, project, tags, request):
```

Comment:
I might put the `request` first, as that is the normal order in Django in my experience.